src/heur_sg.py

```diff
@@ -20,6 +20,16 @@ def __init__(self, of, maxeval, hmax=np.inf, random_descent=False):
         self.hmax = hmax
         self.random_descent = random_descent
 
+        if self.random_descent:
+            self.name = 'SG_RD'
+        else:
+            self.name = 'SG_SD'
+        if self.hmax is 0:
+            self.name = 'SG_RS'
+
+    def get_specs(self):
+        return self.get_name() + '_hmax={}'.format(self.hmax)
```

Comment:
This is only a little step from a nicer `'{}_hmax={}'.format(self.get_name(), self.hmax)`.

Reply:
Sure